frontend/src/component/project/Project/Project.tsx

```diff
@@ -241,6 +242,7 @@ const Project = () => {
                         indicatorColor="primary"
                         textColor="primary"
                         variant="scrollable"
+                        allowScrollButtonsMobile
```

Comment:
all tabs should scroll on mobile

Reply:
Something to look into for other tabs aswell